shard/src/ext/events_cog.py

```diff
@@ -28,6 +32,11 @@ def __init__(self, msg, title, options, votes, exclusive):
         self.exclusive = exclusive
 
 
+class ReactionEventType(IntEnum):
```

Comment:
what does `IntEnum` do, why not just use a class?

Reply:
It easily converts to an Enum. Its just a mapping data type for easily adding stuff to the DB